meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_list.html

```diff
@@ -41,13 +39,13 @@
             {% map_display_points object_list module.settings_instance.polygon %}
         </div>
     {% else %}
-        <div class="module-content">
+        <div class="module-content--light">
             <div class="l-wrapper">
                 <div class="l-center-8">
                     {% if object_list.count > 0 %}
                         <ul class="u-list-reset">
                             {% for object in object_list %}
-                                {% include "meinberlin_kiezkasse/includes/proposal_list_item.html" with object=object %}
+                                {% include "meinberlin_contrib/includes/proposal_list_item.html" with object=object %}
```

Comment:
just out of curiosity. What is the difference between the folder `meinberlin_kiezkasse` and `meinberlin_contrib` ?

Reply:
`meinberlin_kiezkasse`is a module app specific location whereas this template is used in 2 seperate module apps so moved it to contrib so don't have duplicate code